2_FirstComponent.md

```diff
@@ -14,52 +14,6 @@
 
 - - -
 
-##Constructors in CanJS
-
-Before we work with any of the objects in CanJS, it will be helpful for us if we understand can.Construct. Most of the objects in CanJS are derived from can.Construct. can.Construct provides a way to easily use the power of prototypal inheritance without worrying about hooking up all the particulars yourself.
-
-Without going into exhaustive detail[^ConstructDetail], can.Construct contains a few methods we'll encounter frequently in other objects:
-
-[^ConstructDetail]: If you want to go into exhaustive detail on can.Construct, you can consult the [CanJS docs](http://canjs.com/docs/can.Construct.prototype.init.html).
-
-- Prototype
-	- init
-- Static
-	- extend
-
-We'll look at the extend method first.
-
-###The extend method
-can.Construct's `extend` method is used to create "constructor functions". Constructor functions create instances of objects. The extend method can take up to three arguments:
-
-1. name: string
-2. staticProperties: object
-3. instanceProperties: object
-
-The extend method behaves differently depending on the number of arguments you pass it. The name and staticProperties arugments are optional. For example, if you pass it one argument, it will be use the value you pass it to set its instanceProperties. If you pass it two arguments, it use the first to set its staticProperties, and the second to set its instanceProperties. Finally, if you pass in all three arguments, the first will set its name, the second in staticProperties, and the third its instanceProperties.
-
-In the example below, I only want to pass in staticProperties. Therefore, I must call the method as follows:
-
-	can.Construct.extend({
-    	//Static properties here
-    },
-  	//Blank object as second parameter
-  	{});
-
-This pattern will apply to all objects in CanJS that have an extend method.
-
-###The init method
-The `init` method is called whenever a new instance of a can.Construct is created. Init is where the bulk of your initialization code should go. Inside of the init function, the `this` keyword will refer to the new instance, and `this` will contains the arguments passed to the constructor. A common thing to do in init is save the arguments passed into the constructor. An example is below:
-
-    var Person = can.Construct.extend({
-        init: function(first, last) {
-        	this.first = first;
-        	this.last = last;
-    	}
-    });
-
-    var actor = new Person("Abe", "Vigoda");
-
```

Comment:
Even though this is a great explanation of can.Construct, I think it's better suited for the API docs.